R/rdom.R

```diff
@@ -51,14 +50,14 @@ rdom <- function(url, css, all, timeout, filename) {
   st <- attr(res, 'status')
   if (!is.null(st)) stop(paste(res, '\n'))
   p <- if (missing(filename)) {
-    XML::htmlParse(res, asText = TRUE)
+    xml2::read_xml(res, asText = TRUE)
   } else {
-    XML::htmlParse(filename)
+    xml2::read_xml(filename)
   }
```

Comment:
Looks like this `if` statement is no longer necessary since `read_xml()` supports local paths?

Reply:
In other words, I think this could just be `p <-  xml2::read_xml(filename)`

Reply:
Oh.. yes. I'll check this out.